studio/schemas/deskStructure.ts

```diff
@@ -22,13 +22,26 @@ export default (S: StructureBuilder) =>
     .title("Content")
     .items([
       S.listItem()
-        .title("Company Information")
+        .title("Company Details")
         .icon(CaseIcon)
         .child(
-          S.document()
-            .schemaType(companyInfoID)
-            .documentId(companyInfoID)
-            .title("Company Information"),
+          S.list()
+            .title("Company Details")
+            .items([
+              S.listItem()
+                .title("Company Information")
+                .icon(InfoOutlineIcon)
+                .child(
+                  S.document()
+                    .schemaType(companyInfoID)
+                    .documentId(companyInfoID)
+                    .title("Company Information"),
+                ),
+              S.listItem()
+                .title("Offices")
+                .icon(CaseIcon)
```

Comment:
Can we use `HomeIcon` for Offices?

<img width="344" alt="image" src="https://github.com/user-attachments/assets/b14eb4c6-6130-4774-9784-75c90da778ca">

Reply:
Of course!

Reply:
Since we're renaming to Location, what about: PinIcon, EarthAmericasIcon, or EarthGlobeIcon..?